examples/typescript-browser/README.md

```diff
@@ -0,0 +1,38 @@
+# Typescript React demo
+
+## Requirements
+
+Build the base project `/meilisearch-js` as this example uses the locally build bundles from `meilisearch-js` 
+
+```
+$ cd meilisearch-js
+$ yarn build
+```
+
+## Credentials 
+This example uses the following MeiliSearch address: `http://127.0.0.1:7700`. Feel free to change the credentials to meet your MeiliSearch address.
+
+The credentials are written in `src/index.ts`.
+
+## Try out
+
+To try it out you need to follow these steps
+
+### 1. Install dependencies
+
+```bash
+$ yarn
+```
+
+### 2. Build
+
+```bash
+$ yarn build
+```
+
+### 3. Serve
+```bash
+$ http-server public
+```
+
+We used http-server cli to serve the webpage. Feel free to use any tool with the same purpose. It should work.
```

Comment:
Can we remove the "It should work"? Maybe can we replace it by a more confident sentence 😉